### PR TITLE
Gp2 410 sign up login flow

### PR DESCRIPTION
**Pull request by dahfool**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: core/templates/core/page_no_navigation.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+		<div class="bg-blue-deep-80 sign-up--header {% block header_class %}{% endblock %}">
+	        <img class="m-f-auto m-r-auto auto p-t-m" src="{% static 'images/logo.svg' %}" alt="Exporting is Great" id="header-logo-exporting-is-great" width="135">
+	        <div class="sign-up--wrapper">
+	          <div class='form--sign-up text-16'>
+                <div class='sign-up--inner bg-white p-v-s p-h-s'>
+                    {% block inner_content %}<div id="sign-up"></div>{% endblock %}
+                </div>
+                <p class='g-panel'>For any questions, queries or feedback please get in touch with <a class='text-red-80' href="mailto:private@digital.trade.gov.uk" target="_blank" rel="noopener noreferrer">private@digital.trade.gov.uk</a></p>
+              </div>
+            </div>
```
Comment (lewis-coulson-dit) [inline core/templates/core/page_no_navigation.html:15]: indenting a bit off here

File: react-components/tests/components/SocialLoginButtons.test.jsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ afterEach(() => {
   jest.useRealTimers()
 })
 
-test('SocialLoginButtons should render', () => {
+xtest('SocialLoginButtons should render', () => {
```
[the remark this answers is not in the document]
Comment (dahfool) [reply react-components/tests/components/SocialLoginButtons.test.jsx:20]: This test returns a false positive and should be written.

**Review by bobmeredith — APPROVED** [review]
Looks better now.

**Review by lewis-coulson-dit — APPROVED** [review]
LGTM!

**Pull request merged**